modules/nw-multus-bond-cni-object.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-multus-bond-cni-object_{context}"]
+= Configuration for a bond-cni secondary network
+
+The Bond Container Network Interface (Bond-CNI) enables the aggregation of multiple network interfaces into a single logical "bonded" interface within a container, enhancing network redundancy and fault tolerance. Only SR-IOV Virtual Functions (VFs) are supported for bonding with this plugin.
+
+The following object describes the configuration parameters for the bond CNI plugin:
+
+.Bond CNI plugin JSON configuration object
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
+
+
+|`name`
+|`string`
+|The value for the `name` parameter you provided previously for the CNO configuration.
```

Comment:
Also, does CNO need a definition? (or is it a typo of CNI? I am a networking dummy :sweat_smile:)

```suggestion
|The value for the `name` parameter that you provided for the CNO configuration.
```

Reply:
Thanks for picking this up looks like in correct description for this field and this is repeated across all CNIs.